diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter2/kdoc/KdocMethods.kt

```diff
@@ -113,12 +114,13 @@ class KdocMethods(configRules: List<RulesConfig>) : DiktatRule(
         val paramCheckFailed = (missingParameters.isNotEmpty() && !node.isSingleLineGetterOrSetter()) || kDocMissingParameters.isNotEmpty()
         val returnCheckFailed = hasReturnCheckFailed(node, kdocTags)
         val throwsCheckFailed = missingExceptions.isNotEmpty()
+        val checkFunName = isReferenceExpressionWithSameName(node, kdocTags)
```

Comment:
`checkFunName` is not very descriptive. Why not just use `isReferenceExpressionWithSameName` or call this flag `isSimpleFunction` or something simialr